pkgs/servers/varnish/default.nix

```diff
@@ -91,17 +91,17 @@ in
 {
   # EOL (LTS) TBA
   varnish60 = common {
-    version = "6.0.13";
-    hash = "sha256-DcpilfnGnUenIIWYxBU4XFkMZoY+vUK/6wijZ7eIqbo=";
+    version = "6.0.14";
+    hash = "sha256-tZlBf3ppntxxYSufEJ86ot6ujvnbfIyZOu9B3kDJ72k=";
   };
   # EOL 2025-09-15
   varnish76 = common {
```

Comment:
Unrelated, but this will be EOL within 25.05. ALthough we're past the freeze, this should perhaps be removed?